### ISSUE TITLE
Multiple Themes: Improve the Table Block Styles Preview 

### ISSUE THREAD

**Issue by kspilarski** [Automattic/themes#7752]
### What

The preview of the table block Stripes Style does not show the alternating gray colors of the rows. 

### Why

The two table styles Default and Stripes both show the preview of the table with a gray background. When Stripes style is clicked, the preview does not show the alternating gray colors of the rows. 

### How

In the Table block Styles preview, remove the gray background to show the alternating row colors in the Stripes style. 

In Account Settings, changing the Dashboard color scheme does not have any effect on the Table Styles preview.  

**Default:** 

<img width="608" alt="table - default" src="https://github.com/Automattic/wp-calypso/assets/36863633/73854f0a-8c8f-4441-aaa0-83cdb760b77c">

**Stripes**:

<img width="624" alt="table - stripes" src="https://github.com/Automattic/wp-calypso/assets/36863633/a977a27f-0245-47a9-82ef-05e6cc96faa1">

**Comment by cat-og** [comment]
📌 REPRODUCTION RESULTS
- Tested on Simple – Could Not Replicate 
- Tested on Atomic – Could Not Replicate 


📌 FINDINGS/SCREENSHOTS/VIDEO
- This refers to the preview of the Styles options when you hover over 'Default' and 'Stripes' in the settings sidebar, not to the preview of the Table block on the page or Editor itself.
- I could not replicate this on my Atomic or Simple sites; 
 ![Image](https://github.com/Automattic/wp-calypso/assets/86028337/807e36df-1727-4ed1-bb22-c8dcf99cfd83)

However, I did notice that the default color options on my simple site, using the Twenty Twenty-Four made that contrast very difficult to see:

![Image](https://github.com/Automattic/wp-calypso/assets/86028337/f822fb6e-ea95-4b31-93bf-59258260b0f2)


📌 ACTIONS
- Requested author feedback

📌 Message to Author
@kspilarski which theme were you using when you encountered this issue?

**Comment by kspilarski** [comment]
Epi and Sten themes (where gray is the main color), Stripes doesn't show the alternating colors: 

<img width="664" alt="Screenshot 2024-01-02 at 14 41 45" src="https://github.com/Automattic/wp-calypso/assets/36863633/01f01dc0-7bc2-4b02-be25-d5875859941a">


I tested this also using another theme like Organizer, and Stripes show the alternating colors: 

<img width="879" alt="Screenshot 2024-01-02 at 14 38 37" src="https://github.com/Automattic/wp-calypso/assets/36863633/02df54e9-54cf-40e2-861b-c7cd4adb9ebb">

Dara theme also show Stripes with alternating colors, but subtle and not so visible: 

<img width="630" alt="Screenshot 2024-01-02 at 14 40 13" src="https://github.com/Automattic/wp-calypso/assets/36863633/cfc57166-bcd6-4d4c-8be3-9d737d9e8102">

**Comment by liviopv** [comment]
@kspilarski Thanks for the additional context, sorry for the delayed reply.

It sounds more like a bug than a feature request, and, there is an existing issue here https://github.com/WordPress/gutenberg/issues/60091. Would you be open to restructuring the original report based on that and listing the themes where it doesn't work? 

I left a comment here https://github.com/WordPress/gutenberg/issues/60091#issuecomment-2023016635 to add context based on what you reported, but please jump to share more as needed, just remember it's a public community-led repository.

Also, there is a workaround to ensure contrast in the Stripe styles:

1. Appearance > Editor > Styles
2. Blocks > Table Block
3. Style Variations > Stripes
4. Adjust the background color

**Comment by rickmgithub** [comment]
Re-pinged OP directly to check.

**Comment by mrfoxtalbot** [comment]
I have followed up with the conversation in https://github.com/WordPress/gutenberg/issues/60091#issuecomment-2023016635

We used to have a label for "Needs upstream Fix" but I cannot find it. I think it would make sense to continue using that label.

**Comment by kspilarski** [comment]
> It sounds more like a bug than a feature request, and, there is an existing issue here https://github.com/WordPress/gutenberg/issues/60091. Would you be open to restructuring the original report based on that and listing the themes where it doesn't work?

- Epi and Sten themes (where gray is the main color), Stripes doesn't show the alternating colors

**Comment by mrfoxtalbot** [comment]
As per [this comment](https://github.com/WordPress/gutenberg/issues/60091#issuecomment-2063014098) @t-hamano this issue seems to be theme-specific. I will transfer this to the themes repo.

**Comment by qugkyp-vudgaz-2maQba** [comment]
Okay
